### PR TITLE
[Update] Manually Enable Elastic IP on Your Linode

### PR DESCRIPTION
**Pull request by rsyracuse**

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 3b7081602cde45b0f00895daa1c07e94eea41131

https://deploy-preview-4003--nostalgic-ptolemy-b01ab8.netlify.app

**Review by christinepuk — CHANGES_REQUESTED** [review]
We need the `[UNPADDED_DC_ID]` on line 406 as well so it should be 
```
 set large-community 65[DC_ID]5:[UNPADDED_DC_ID]:1
```

**Review by christinepuk — CHANGES_REQUESTED** [review]
should be 
```
set large-community 1234:5:1
route-map secondary permit 10
set large-community 1234:5:2
```

**Review by christinepuk — APPROVED** [review]
looks great! :tada:

**Review by andystevensname — APPROVED** [review]

File: docs/guides/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md [hunk 1]
```diff
@@ -402,9 +403,9 @@ address-family ipv4 unicast
   redistribute static
 exit-address-family
 route-map primary permit 10
-set large-community 65[DC_ID]:[DC_ID]:1
+set large-community 1234:5:1
 route-map secondary permit 10
-set large-community 65[DC_ID]5:[DC_ID]:2
+set large-community 1234:5:2
```
Comment (christinepuk) [inline docs/guides/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md:408]: I am sorry @rsyracuse  we need to change this again, it should be 
```
set large-community 63949:1:1 
route-map secondary permit 10
set large-community  63949:1:2
```

**Pull request merged**